### ISSUE TITLE
no-wildcard-imports does not seem to work

### ISSUE THREAD

**Issue by gavvvr** [pinterest/ktlint#1286]
Consider the following simple example:

```kotlin
package com.example
// notice the wildcard import below
import org.springframework.boot.autoconfigure.*
import org.springframework.boot.runApplication

@SpringBootApplication
class DemoApplication

fun main(args: Array<String>) {    
    runApplication<DemoApplication>(*args)
}
```

## Expected Behavior

I expect the `no-wildcard-imports` rule to be triggered

## Observed Behavior

Silent exit with zero return-code

## Steps to Reproduce

- Create **DemoApplication.kt** with the content above
- Execute `ktlint DemoApplication.kt`

## Your Environment

* Version of ktlint used: 0.43.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): bare `ktlint` command
* Operating System and version: MacOS 11.5.1

**Comment by JanCizmar** [comment]
Downgrading to 0.42.1 solves this issue.

**Comment by paul-dingemans** [comment]
For me it does work with example given above. Note that `no-wildcard-imports` is an experimental rule and you need to specify the command line flag `--experimental` to enable it. 

If problem persist, please specify the exact command that you execute. And, please verify that the rule is not disabled in `.editconfig`.

**Comment by gavvvr** [comment]
Thanks @paul-dingemans. Now it works. The problem was with the presence of `.editconfig` which explicitly disabled the rule. `--debug` helped me to discover it. By the way, `--experimental` is not required for `no-wildcard-imports` rule to be triggered.

@JanCizmar as I said, it was my fault with `.editconfig`. I also tried 0.42.1, but did not notice any difference between versions
